{"Timestamp":"2023-02-04T00:00:01.9271633+03:30","Level":"Information","MessageTemplate":"Now listening on: {address}","Properties":{"address":"http://localhost:5000","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime","MachineName":"LAPTOP-TSMJLLUF","ProcessId":13632,"ThreadId":6,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:00:01.9362012+03:30","Level":"Information","MessageTemplate":"Application started. Press Ctrl+C to shut down.","Properties":{"SourceContext":"Microsoft.Hosting.Lifetime","MachineName":"LAPTOP-TSMJLLUF","ProcessId":13632,"ThreadId":6,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:00:01.9365353+03:30","Level":"Information","MessageTemplate":"Hosting environment: {envName}","Properties":{"envName":"Development","SourceContext":"Microsoft.Hosting.Lifetime","MachineName":"LAPTOP-TSMJLLUF","ProcessId":13632,"ThreadId":6,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:00:01.9368535+03:30","Level":"Information","MessageTemplate":"Content root path: {contentRoot}","Properties":{"contentRoot":"D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.RestApi\\","SourceContext":"Microsoft.Hosting.Lifetime","MachineName":"LAPTOP-TSMJLLUF","ProcessId":13632,"ThreadId":6,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:14:55.2287386+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":23352,"ThreadId":10,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:15:01.7489365+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":23352,"ThreadId":10,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:15:05.0526766+03:30","Level":"Error","MessageTemplate":"Failed executing DbCommand ({elapsed}ms) [Parameters=[{parameters}], CommandType='{commandType}', CommandTimeout='{commandTimeout}']{newLine}{commandText}","Properties":{"elapsed":"10","parameters":"@p0='?' (Size = 450), @p1='?' (DbType = Int32), @p2='?' (Size = 4000), @p3='?' (DbType = DateTime2), @p4='?' (Size = 100), @p5='?' (DbType = Boolean), @p6='?' (Size = 100), @p7='?' (DbType = Boolean), @p8='?' (DbType = DateTimeOffset), @p9='?' (Size = 10), @p10='?' (Size = 50), @p11='?' (Size = 4000), @p12='?' (Size = 4000), @p13='?' (Size = 450), @p14='?' (DbType = Boolean), @p15='?' (Size = 50), @p16='?' (Size = 5), @p17='?' (Size = 11)","commandType":"Text","commandTimeout":30,"newLine":"\r\n","commandText":"SET NOCOUNT ON;\r\nINSERT INTO [ApplicationUsers] ([Id], [AccessFailedCount], [ConcurrencyStamp], [CreationDate], [FirstName], [IsActive], [LastName], [LockoutEnabled], [LockoutEnd], [NationalCode], [NormalizedUserName], [PasswordHash], [SecurityStamp], [TenantId], [TwoFactorEnabled], [UserName], [Mobile_CountryCallingCode], [Mobile_Number])\r\nVALUES (@p0, @p1, @p2, @p3, @p4, @p5, @p6, @p7, @p8, @p9, @p10, @p11, @p12, @p13, @p14, @p15, @p16, @p17);","EventId":{"Id":20102,"Name":"Microsoft.EntityFrameworkCore.Database.Command.CommandError"},"SourceContext":"Microsoft.EntityFrameworkCore.Database.Command","MachineName":"LAPTOP-TSMJLLUF","ProcessId":23352,"ThreadId":14,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:15:05.0711153+03:30","Level":"Error","MessageTemplate":"An exception occurred in the database while saving changes for context type '{contextType}'.{newline}{error}","Exception":"Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot insert the value NULL into column 'TenantId', table 'PhoenixDb.dbo.ApplicationUsers'; column does not allow nulls. INSERT fails.\r\nThe statement has been terminated.\r\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__188_0(Task`1 result)\r\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\r\n   at System.Threading.Tasks.Task.<>c.<.cctor>b__272_0(Object obj)\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location ---\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\nClientConnectionId:5a736904-4e4e-45f2-bc32-b4897a13eaa7\r\nError Number:515,State:2,Class:16\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)","Properties":{"contextType":"Phoenix.DataSources.Infrastructures.DBContexts.EFDataContext","newline":"\r\n","error":"Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot insert the value NULL into column 'TenantId', table 'PhoenixDb.dbo.ApplicationUsers'; column does not allow nulls. INSERT fails.\r\nThe statement has been terminated.\r\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__188_0(Task`1 result)\r\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\r\n   at System.Threading.Tasks.Task.<>c.<.cctor>b__272_0(Object obj)\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location ---\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\nClientConnectionId:5a736904-4e4e-45f2-bc32-b4897a13eaa7\r\nError Number:515,State:2,Class:16\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)","EventId":{"Id":10000,"Name":"Microsoft.EntityFrameworkCore.Update.SaveChangesFailed"},"SourceContext":"Microsoft.EntityFrameworkCore.Update","MachineName":"LAPTOP-TSMJLLUF","ProcessId":23352,"ThreadId":14,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:15:05.3293705+03:30","Level":"Fatal","MessageTemplate":"Unhandled exception","Exception":"System.AggregateException: One or more errors occurred. (An error occurred while saving the entity changes. See the inner exception for details.)\r\n ---> Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot insert the value NULL into column 'TenantId', table 'PhoenixDb.dbo.ApplicationUsers'; column does not allow nulls. INSERT fails.\r\nThe statement has been terminated.\r\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__188_0(Task`1 result)\r\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\r\n   at System.Threading.Tasks.Task.<>c.<.cctor>b__272_0(Object obj)\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location ---\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\nClientConnectionId:5a736904-4e4e-45f2-bc32-b4897a13eaa7\r\nError Number:515,State:2,Class:16\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.AspNetCore.Identity.EntityFrameworkCore.UserStore`9.CreateAsync(TUser user, CancellationToken cancellationToken)\r\n   at Microsoft.AspNetCore.Identity.UserManager`1.CreateAsync(TUser user)\r\n   at Microsoft.AspNetCore.Identity.UserManager`1.CreateAsync(TUser user, String password)\r\n   at Phoenix.Application.Services.ApplicationUsers.ApplicationUserAppService.AddUser(AddApplicationUserDto dto) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Services\\Services\\ApplicationUsers\\ApplicationUserAppService.cs:line 51\r\n   at Phoenix.Infrastructure.SeedDatas.AdminSeedDataConfiguration.UseSeedData(IApplicationBuilder applicationBuilder, IConfiguration configuration) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\SeedDatas\\AdminSeedDataConfiguration.cs:line 42\r\n   --- End of inner exception stack trace ---\r\n   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)\r\n   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)\r\n   at System.Threading.Tasks.Task.Wait()\r\n   at Phoenix.Infrastructure.Startup.UseInfrastructure(IApplicationBuilder builder, IConfiguration config) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\Startup.cs:line 109\r\n   at Program.<Main>$(String[] args) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.RestApi\\Program.cs:line 38","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":23352,"ThreadId":10,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:15:05.5017166+03:30","Level":"Information","MessageTemplate":"Server Shutting down...","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":23352,"ThreadId":10,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:15:42.5394234+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":12440,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:15:45.8440989+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":12440,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:15:48.0203000+03:30","Level":"Error","MessageTemplate":"Failed executing DbCommand ({elapsed}ms) [Parameters=[{parameters}], CommandType='{commandType}', CommandTimeout='{commandTimeout}']{newLine}{commandText}","Properties":{"elapsed":"8","parameters":"@p0='?' (Size = 450), @p1='?' (DbType = Int32), @p2='?' (Size = 4000), @p3='?' (DbType = DateTime2), @p4='?' (Size = 100), @p5='?' (DbType = Boolean), @p6='?' (Size = 100), @p7='?' (DbType = Boolean), @p8='?' (DbType = DateTimeOffset), @p9='?' (Size = 10), @p10='?' (Size = 50), @p11='?' (Size = 4000), @p12='?' (Size = 4000), @p13='?' (Size = 450), @p14='?' (DbType = Boolean), @p15='?' (Size = 50), @p16='?' (Size = 5), @p17='?' (Size = 11)","commandType":"Text","commandTimeout":30,"newLine":"\r\n","commandText":"SET NOCOUNT ON;\r\nINSERT INTO [ApplicationUsers] ([Id], [AccessFailedCount], [ConcurrencyStamp], [CreationDate], [FirstName], [IsActive], [LastName], [LockoutEnabled], [LockoutEnd], [NationalCode], [NormalizedUserName], [PasswordHash], [SecurityStamp], [TenantId], [TwoFactorEnabled], [UserName], [Mobile_CountryCallingCode], [Mobile_Number])\r\nVALUES (@p0, @p1, @p2, @p3, @p4, @p5, @p6, @p7, @p8, @p9, @p10, @p11, @p12, @p13, @p14, @p15, @p16, @p17);","EventId":{"Id":20102,"Name":"Microsoft.EntityFrameworkCore.Database.Command.CommandError"},"SourceContext":"Microsoft.EntityFrameworkCore.Database.Command","MachineName":"LAPTOP-TSMJLLUF","ProcessId":12440,"ThreadId":14,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:15:48.0483842+03:30","Level":"Error","MessageTemplate":"An exception occurred in the database while saving changes for context type '{contextType}'.{newline}{error}","Exception":"Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot insert the value NULL into column 'TenantId', table 'PhoenixDb.dbo.ApplicationUsers'; column does not allow nulls. INSERT fails.\r\nThe statement has been terminated.\r\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__188_0(Task`1 result)\r\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\r\n   at System.Threading.Tasks.Task.<>c.<.cctor>b__272_0(Object obj)\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location ---\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\nClientConnectionId:0a75cb0c-555b-4e5d-877f-1472ea564406\r\nError Number:515,State:2,Class:16\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)","Properties":{"contextType":"Phoenix.DataSources.Infrastructures.DBContexts.EFDataContext","newline":"\r\n","error":"Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot insert the value NULL into column 'TenantId', table 'PhoenixDb.dbo.ApplicationUsers'; column does not allow nulls. INSERT fails.\r\nThe statement has been terminated.\r\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__188_0(Task`1 result)\r\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\r\n   at System.Threading.Tasks.Task.<>c.<.cctor>b__272_0(Object obj)\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location ---\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\nClientConnectionId:0a75cb0c-555b-4e5d-877f-1472ea564406\r\nError Number:515,State:2,Class:16\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)","EventId":{"Id":10000,"Name":"Microsoft.EntityFrameworkCore.Update.SaveChangesFailed"},"SourceContext":"Microsoft.EntityFrameworkCore.Update","MachineName":"LAPTOP-TSMJLLUF","ProcessId":12440,"ThreadId":14,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:15:48.3569166+03:30","Level":"Fatal","MessageTemplate":"Unhandled exception","Exception":"System.AggregateException: One or more errors occurred. (An error occurred while saving the entity changes. See the inner exception for details.)\r\n ---> Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot insert the value NULL into column 'TenantId', table 'PhoenixDb.dbo.ApplicationUsers'; column does not allow nulls. INSERT fails.\r\nThe statement has been terminated.\r\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__188_0(Task`1 result)\r\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\r\n   at System.Threading.Tasks.Task.<>c.<.cctor>b__272_0(Object obj)\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location ---\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\nClientConnectionId:0a75cb0c-555b-4e5d-877f-1472ea564406\r\nError Number:515,State:2,Class:16\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.AspNetCore.Identity.EntityFrameworkCore.UserStore`9.CreateAsync(TUser user, CancellationToken cancellationToken)\r\n   at Microsoft.AspNetCore.Identity.UserManager`1.CreateAsync(TUser user)\r\n   at Microsoft.AspNetCore.Identity.UserManager`1.CreateAsync(TUser user, String password)\r\n   at Phoenix.Application.Services.ApplicationUsers.ApplicationUserAppService.AddUser(AddApplicationUserDto dto) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Services\\Services\\ApplicationUsers\\ApplicationUserAppService.cs:line 51\r\n   at Phoenix.Infrastructure.SeedDatas.AdminSeedDataConfiguration.UseSeedData(IApplicationBuilder applicationBuilder, IConfiguration configuration) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\SeedDatas\\AdminSeedDataConfiguration.cs:line 42\r\n   --- End of inner exception stack trace ---\r\n   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)\r\n   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)\r\n   at System.Threading.Tasks.Task.Wait()\r\n   at Phoenix.Infrastructure.Startup.UseInfrastructure(IApplicationBuilder builder, IConfiguration config) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\Startup.cs:line 109\r\n   at Program.<Main>$(String[] args) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.RestApi\\Program.cs:line 38","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":12440,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:15:48.4747888+03:30","Level":"Information","MessageTemplate":"Server Shutting down...","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":12440,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:33:19.6718231+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":28880,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:33:23.2139707+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":28880,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:33:55.2231156+03:30","Level":"Fatal","MessageTemplate":"Unhandled exception","Exception":"System.AggregateException: One or more errors occurred. (Exception of type 'Phoenix.Application.Services.ApplicationUsers.Exceptions.InvalidMobileNumberException' was thrown.)\r\n ---> Phoenix.Application.Services.ApplicationUsers.Exceptions.InvalidMobileNumberException: Exception of type 'Phoenix.Application.Services.ApplicationUsers.Exceptions.InvalidMobileNumberException' was thrown.\r\n   at Phoenix.Application.Services.ApplicationUsers.ApplicationUserAppService.GaurdAgainstInvalidMobileNumber(String mobileNumber) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Services\\Services\\ApplicationUsers\\ApplicationUserAppService.cs:line 250\r\n   at Phoenix.Application.Services.ApplicationUsers.ApplicationUserAppService.AddSuperAdminUser(ApplicationUser adminUser) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Services\\Services\\ApplicationUsers\\ApplicationUserAppService.cs:line 36\r\n   at Phoenix.Infrastructure.SeedDatas.AdminSeedDataConfiguration.UseSeedData(IApplicationBuilder applicationBuilder, IConfiguration configuration) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\SeedDatas\\AdminSeedDataConfiguration.cs:line 43\r\n   --- End of inner exception stack trace ---\r\n   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)\r\n   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)\r\n   at System.Threading.Tasks.Task.Wait()\r\n   at Phoenix.Infrastructure.Startup.UseInfrastructure(IApplicationBuilder builder, IConfiguration config) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\Startup.cs:line 109\r\n   at Program.<Main>$(String[] args) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.RestApi\\Program.cs:line 38","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":28880,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:33:55.3869449+03:30","Level":"Information","MessageTemplate":"Server Shutting down...","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":28880,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:34:30.2480151+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":17036,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:34:33.4308861+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":17036,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:34:55.2329337+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":928,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:34:58.3756500+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":928,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:35:11.3461692+03:30","Level":"Fatal","MessageTemplate":"Unhandled exception","Exception":"System.AggregateException: One or more errors occurred. (Exception of type 'Phoenix.Application.Services.ApplicationUsers.Exceptions.InvalidMobileNumberException' was thrown.)\r\n ---> Phoenix.Application.Services.ApplicationUsers.Exceptions.InvalidMobileNumberException: Exception of type 'Phoenix.Application.Services.ApplicationUsers.Exceptions.InvalidMobileNumberException' was thrown.\r\n   at Phoenix.Application.Services.ApplicationUsers.ApplicationUserAppService.GaurdAgainstInvalidMobileNumber(String mobileNumber) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Services\\Services\\ApplicationUsers\\ApplicationUserAppService.cs:line 250\r\n   at Phoenix.Application.Services.ApplicationUsers.ApplicationUserAppService.AddSuperAdminUser(ApplicationUser adminUser) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Services\\Services\\ApplicationUsers\\ApplicationUserAppService.cs:line 36\r\n   at Phoenix.Infrastructure.SeedDatas.AdminSeedDataConfiguration.UseSeedData(IApplicationBuilder applicationBuilder, IConfiguration configuration) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\SeedDatas\\AdminSeedDataConfiguration.cs:line 43\r\n   --- End of inner exception stack trace ---\r\n   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)\r\n   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)\r\n   at System.Threading.Tasks.Task.Wait()\r\n   at Phoenix.Infrastructure.Startup.UseInfrastructure(IApplicationBuilder builder, IConfiguration config) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\Startup.cs:line 109\r\n   at Program.<Main>$(String[] args) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.RestApi\\Program.cs:line 38","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":928,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:35:11.3926506+03:30","Level":"Information","MessageTemplate":"Server Shutting down...","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":928,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:35:45.7632356+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":10424,"ThreadId":14,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:35:48.9304916+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":10424,"ThreadId":14,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:36:28.2784067+03:30","Level":"Fatal","MessageTemplate":"Unhandled exception","Exception":"System.AggregateException: One or more errors occurred. (Exception of type 'Phoenix.Application.Services.ApplicationUsers.Exceptions.InvalidMobileNumberException' was thrown.)\r\n ---> Phoenix.Application.Services.ApplicationUsers.Exceptions.InvalidMobileNumberException: Exception of type 'Phoenix.Application.Services.ApplicationUsers.Exceptions.InvalidMobileNumberException' was thrown.\r\n   at Phoenix.Application.Services.ApplicationUsers.ApplicationUserAppService.GaurdAgainstInvalidMobileNumber(String mobileNumber) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Services\\Services\\ApplicationUsers\\ApplicationUserAppService.cs:line 250\r\n   at Phoenix.Application.Services.ApplicationUsers.ApplicationUserAppService.AddSuperAdminUser(ApplicationUser adminUser) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Services\\Services\\ApplicationUsers\\ApplicationUserAppService.cs:line 36\r\n   at Phoenix.Infrastructure.SeedDatas.AdminSeedDataConfiguration.UseSeedData(IApplicationBuilder applicationBuilder, IConfiguration configuration) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\SeedDatas\\AdminSeedDataConfiguration.cs:line 43\r\n   --- End of inner exception stack trace ---\r\n   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)\r\n   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)\r\n   at System.Threading.Tasks.Task.Wait()\r\n   at Phoenix.Infrastructure.Startup.UseInfrastructure(IApplicationBuilder builder, IConfiguration config) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\Startup.cs:line 109\r\n   at Program.<Main>$(String[] args) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.RestApi\\Program.cs:line 38","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":10424,"ThreadId":14,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:36:28.3313306+03:30","Level":"Information","MessageTemplate":"Server Shutting down...","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":10424,"ThreadId":14,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:36:47.8507909+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":16544,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:36:51.1188272+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":16544,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:36:58.2228222+03:30","Level":"Fatal","MessageTemplate":"Unhandled exception","Exception":"System.AggregateException: One or more errors occurred. (Exception of type 'Phoenix.Application.Services.ApplicationUsers.Exceptions.InvalidMobileNumberException' was thrown.)\r\n ---> Phoenix.Application.Services.ApplicationUsers.Exceptions.InvalidMobileNumberException: Exception of type 'Phoenix.Application.Services.ApplicationUsers.Exceptions.InvalidMobileNumberException' was thrown.\r\n   at Phoenix.Application.Services.ApplicationUsers.ApplicationUserAppService.GaurdAgainstInvalidMobileNumber(String mobileNumber) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Services\\Services\\ApplicationUsers\\ApplicationUserAppService.cs:line 250\r\n   at Phoenix.Application.Services.ApplicationUsers.ApplicationUserAppService.AddSuperAdminUser(ApplicationUser adminUser) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Services\\Services\\ApplicationUsers\\ApplicationUserAppService.cs:line 36\r\n   at Phoenix.Infrastructure.SeedDatas.AdminSeedDataConfiguration.UseSeedData(IApplicationBuilder applicationBuilder, IConfiguration configuration) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\SeedDatas\\AdminSeedDataConfiguration.cs:line 43\r\n   --- End of inner exception stack trace ---\r\n   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)\r\n   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)\r\n   at System.Threading.Tasks.Task.Wait()\r\n   at Phoenix.Infrastructure.Startup.UseInfrastructure(IApplicationBuilder builder, IConfiguration config) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\Startup.cs:line 109\r\n   at Program.<Main>$(String[] args) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.RestApi\\Program.cs:line 38","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":16544,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:36:58.3017395+03:30","Level":"Information","MessageTemplate":"Server Shutting down...","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":16544,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:38:06.4928830+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":11116,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:38:09.8818950+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":11116,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:38:17.8506260+03:30","Level":"Error","MessageTemplate":"Failed executing DbCommand ({elapsed}ms) [Parameters=[{parameters}], CommandType='{commandType}', CommandTimeout='{commandTimeout}']{newLine}{commandText}","Properties":{"elapsed":"23","parameters":"@p0='?' (Size = 450), @p1='?' (DbType = Int32), @p2='?' (Size = 4000), @p3='?' (DbType = DateTime2), @p4='?' (Size = 100), @p5='?' (DbType = Boolean), @p6='?' (Size = 100), @p7='?' (DbType = Boolean), @p8='?' (DbType = DateTimeOffset), @p9='?' (Size = 10), @p10='?' (Size = 50), @p11='?' (Size = 4000), @p12='?' (Size = 4000), @p13='?' (Size = 450), @p14='?' (DbType = Boolean), @p15='?' (Size = 50), @p16='?' (Size = 5), @p17='?' (Size = 11)","commandType":"Text","commandTimeout":30,"newLine":"\r\n","commandText":"SET NOCOUNT ON;\r\nINSERT INTO [ApplicationUsers] ([Id], [AccessFailedCount], [ConcurrencyStamp], [CreationDate], [FirstName], [IsActive], [LastName], [LockoutEnabled], [LockoutEnd], [NationalCode], [NormalizedUserName], [PasswordHash], [SecurityStamp], [TenantId], [TwoFactorEnabled], [UserName], [Mobile_CountryCallingCode], [Mobile_Number])\r\nVALUES (@p0, @p1, @p2, @p3, @p4, @p5, @p6, @p7, @p8, @p9, @p10, @p11, @p12, @p13, @p14, @p15, @p16, @p17);","EventId":{"Id":20102,"Name":"Microsoft.EntityFrameworkCore.Database.Command.CommandError"},"SourceContext":"Microsoft.EntityFrameworkCore.Database.Command","MachineName":"LAPTOP-TSMJLLUF","ProcessId":11116,"ThreadId":22,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:38:17.8996939+03:30","Level":"Error","MessageTemplate":"An exception occurred in the database while saving changes for context type '{contextType}'.{newline}{error}","Exception":"Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot insert the value NULL into column 'TenantId', table 'PhoenixDb.dbo.ApplicationUsers'; column does not allow nulls. INSERT fails.\r\nThe statement has been terminated.\r\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__188_0(Task`1 result)\r\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\r\n   at System.Threading.Tasks.Task.<>c.<.cctor>b__272_0(Object obj)\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location ---\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\nClientConnectionId:a09a0437-b7d4-4040-9571-608011df8cd1\r\nError Number:515,State:2,Class:16\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)","Properties":{"contextType":"Phoenix.DataSources.Infrastructures.DBContexts.EFDataContext","newline":"\r\n","error":"Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot insert the value NULL into column 'TenantId', table 'PhoenixDb.dbo.ApplicationUsers'; column does not allow nulls. INSERT fails.\r\nThe statement has been terminated.\r\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__188_0(Task`1 result)\r\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\r\n   at System.Threading.Tasks.Task.<>c.<.cctor>b__272_0(Object obj)\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location ---\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\nClientConnectionId:a09a0437-b7d4-4040-9571-608011df8cd1\r\nError Number:515,State:2,Class:16\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)","EventId":{"Id":10000,"Name":"Microsoft.EntityFrameworkCore.Update.SaveChangesFailed"},"SourceContext":"Microsoft.EntityFrameworkCore.Update","MachineName":"LAPTOP-TSMJLLUF","ProcessId":11116,"ThreadId":22,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:38:18.1200973+03:30","Level":"Fatal","MessageTemplate":"Unhandled exception","Exception":"System.AggregateException: One or more errors occurred. (An error occurred while saving the entity changes. See the inner exception for details.)\r\n ---> Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot insert the value NULL into column 'TenantId', table 'PhoenixDb.dbo.ApplicationUsers'; column does not allow nulls. INSERT fails.\r\nThe statement has been terminated.\r\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__188_0(Task`1 result)\r\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\r\n   at System.Threading.Tasks.Task.<>c.<.cctor>b__272_0(Object obj)\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location ---\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\nClientConnectionId:a09a0437-b7d4-4040-9571-608011df8cd1\r\nError Number:515,State:2,Class:16\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.AspNetCore.Identity.EntityFrameworkCore.UserStore`9.CreateAsync(TUser user, CancellationToken cancellationToken)\r\n   at Microsoft.AspNetCore.Identity.UserManager`1.CreateAsync(TUser user)\r\n   at Microsoft.AspNetCore.Identity.UserManager`1.CreateAsync(TUser user, String password)\r\n   at Phoenix.Application.Services.ApplicationUsers.ApplicationUserAppService.AddSuperAdminUser(ApplicationUser adminUser) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Services\\Services\\ApplicationUsers\\ApplicationUserAppService.cs:line 52\r\n   at Phoenix.Infrastructure.SeedDatas.AdminSeedDataConfiguration.UseSeedData(IApplicationBuilder applicationBuilder, IConfiguration configuration) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\SeedDatas\\AdminSeedDataConfiguration.cs:line 43\r\n   --- End of inner exception stack trace ---\r\n   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)\r\n   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)\r\n   at System.Threading.Tasks.Task.Wait()\r\n   at Phoenix.Infrastructure.Startup.UseInfrastructure(IApplicationBuilder builder, IConfiguration config) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\Startup.cs:line 109\r\n   at Program.<Main>$(String[] args) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.RestApi\\Program.cs:line 38","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":11116,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:38:18.2013839+03:30","Level":"Information","MessageTemplate":"Server Shutting down...","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":11116,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:38:31.8354929+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":19648,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:38:35.4094524+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":19648,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:38:39.5529495+03:30","Level":"Error","MessageTemplate":"Failed executing DbCommand ({elapsed}ms) [Parameters=[{parameters}], CommandType='{commandType}', CommandTimeout='{commandTimeout}']{newLine}{commandText}","Properties":{"elapsed":"15","parameters":"@p0='?' (Size = 450), @p1='?' (DbType = Int32), @p2='?' (Size = 4000), @p3='?' (DbType = DateTime2), @p4='?' (Size = 100), @p5='?' (DbType = Boolean), @p6='?' (Size = 100), @p7='?' (DbType = Boolean), @p8='?' (DbType = DateTimeOffset), @p9='?' (Size = 10), @p10='?' (Size = 50), @p11='?' (Size = 4000), @p12='?' (Size = 4000), @p13='?' (Size = 450), @p14='?' (DbType = Boolean), @p15='?' (Size = 50), @p16='?' (Size = 5), @p17='?' (Size = 11)","commandType":"Text","commandTimeout":30,"newLine":"\r\n","commandText":"SET NOCOUNT ON;\r\nINSERT INTO [ApplicationUsers] ([Id], [AccessFailedCount], [ConcurrencyStamp], [CreationDate], [FirstName], [IsActive], [LastName], [LockoutEnabled], [LockoutEnd], [NationalCode], [NormalizedUserName], [PasswordHash], [SecurityStamp], [TenantId], [TwoFactorEnabled], [UserName], [Mobile_CountryCallingCode], [Mobile_Number])\r\nVALUES (@p0, @p1, @p2, @p3, @p4, @p5, @p6, @p7, @p8, @p9, @p10, @p11, @p12, @p13, @p14, @p15, @p16, @p17);","EventId":{"Id":20102,"Name":"Microsoft.EntityFrameworkCore.Database.Command.CommandError"},"SourceContext":"Microsoft.EntityFrameworkCore.Database.Command","MachineName":"LAPTOP-TSMJLLUF","ProcessId":19648,"ThreadId":10,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:38:39.5907182+03:30","Level":"Error","MessageTemplate":"An exception occurred in the database while saving changes for context type '{contextType}'.{newline}{error}","Exception":"Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot insert the value NULL into column 'TenantId', table 'PhoenixDb.dbo.ApplicationUsers'; column does not allow nulls. INSERT fails.\r\nThe statement has been terminated.\r\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__188_0(Task`1 result)\r\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\r\n   at System.Threading.Tasks.Task.<>c.<.cctor>b__272_0(Object obj)\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location ---\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\nClientConnectionId:8a73812a-7c22-4975-85b0-62ae8f441a9c\r\nError Number:515,State:2,Class:16\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)","Properties":{"contextType":"Phoenix.DataSources.Infrastructures.DBContexts.EFDataContext","newline":"\r\n","error":"Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot insert the value NULL into column 'TenantId', table 'PhoenixDb.dbo.ApplicationUsers'; column does not allow nulls. INSERT fails.\r\nThe statement has been terminated.\r\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__188_0(Task`1 result)\r\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\r\n   at System.Threading.Tasks.Task.<>c.<.cctor>b__272_0(Object obj)\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location ---\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\nClientConnectionId:8a73812a-7c22-4975-85b0-62ae8f441a9c\r\nError Number:515,State:2,Class:16\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)","EventId":{"Id":10000,"Name":"Microsoft.EntityFrameworkCore.Update.SaveChangesFailed"},"SourceContext":"Microsoft.EntityFrameworkCore.Update","MachineName":"LAPTOP-TSMJLLUF","ProcessId":19648,"ThreadId":10,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:38:39.9442256+03:30","Level":"Fatal","MessageTemplate":"Unhandled exception","Exception":"System.AggregateException: One or more errors occurred. (An error occurred while saving the entity changes. See the inner exception for details.)\r\n ---> Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot insert the value NULL into column 'TenantId', table 'PhoenixDb.dbo.ApplicationUsers'; column does not allow nulls. INSERT fails.\r\nThe statement has been terminated.\r\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__188_0(Task`1 result)\r\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\r\n   at System.Threading.Tasks.Task.<>c.<.cctor>b__272_0(Object obj)\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location ---\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\nClientConnectionId:8a73812a-7c22-4975-85b0-62ae8f441a9c\r\nError Number:515,State:2,Class:16\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.AspNetCore.Identity.EntityFrameworkCore.UserStore`9.CreateAsync(TUser user, CancellationToken cancellationToken)\r\n   at Microsoft.AspNetCore.Identity.UserManager`1.CreateAsync(TUser user)\r\n   at Microsoft.AspNetCore.Identity.UserManager`1.CreateAsync(TUser user, String password)\r\n   at Phoenix.Application.Services.ApplicationUsers.ApplicationUserAppService.AddSuperAdminUser(ApplicationUser adminUser) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Services\\Services\\ApplicationUsers\\ApplicationUserAppService.cs:line 52\r\n   at Phoenix.Infrastructure.SeedDatas.AdminSeedDataConfiguration.UseSeedData(IApplicationBuilder applicationBuilder, IConfiguration configuration) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\SeedDatas\\AdminSeedDataConfiguration.cs:line 43\r\n   --- End of inner exception stack trace ---\r\n   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)\r\n   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)\r\n   at System.Threading.Tasks.Task.Wait()\r\n   at Phoenix.Infrastructure.Startup.UseInfrastructure(IApplicationBuilder builder, IConfiguration config) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\Startup.cs:line 109\r\n   at Program.<Main>$(String[] args) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.RestApi\\Program.cs:line 38","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":19648,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:38:40.1029841+03:30","Level":"Information","MessageTemplate":"Server Shutting down...","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":19648,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:39:38.1049152+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":31520,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:39:41.7704790+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":31520,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:39:44.1448915+03:30","Level":"Error","MessageTemplate":"Failed executing DbCommand ({elapsed}ms) [Parameters=[{parameters}], CommandType='{commandType}', CommandTimeout='{commandTimeout}']{newLine}{commandText}","Properties":{"elapsed":"17","parameters":"@p0='?' (Size = 450), @p1='?' (DbType = Int32), @p2='?' (Size = 4000), @p3='?' (DbType = DateTime2), @p4='?' (Size = 100), @p5='?' (DbType = Boolean), @p6='?' (Size = 100), @p7='?' (DbType = Boolean), @p8='?' (DbType = DateTimeOffset), @p9='?' (Size = 10), @p10='?' (Size = 50), @p11='?' (Size = 4000), @p12='?' (Size = 4000), @p13='?' (Size = 450), @p14='?' (DbType = Boolean), @p15='?' (Size = 50), @p16='?' (Size = 5), @p17='?' (Size = 11)","commandType":"Text","commandTimeout":30,"newLine":"\r\n","commandText":"SET NOCOUNT ON;\r\nINSERT INTO [ApplicationUsers] ([Id], [AccessFailedCount], [ConcurrencyStamp], [CreationDate], [FirstName], [IsActive], [LastName], [LockoutEnabled], [LockoutEnd], [NationalCode], [NormalizedUserName], [PasswordHash], [SecurityStamp], [TenantId], [TwoFactorEnabled], [UserName], [Mobile_CountryCallingCode], [Mobile_Number])\r\nVALUES (@p0, @p1, @p2, @p3, @p4, @p5, @p6, @p7, @p8, @p9, @p10, @p11, @p12, @p13, @p14, @p15, @p16, @p17);","EventId":{"Id":20102,"Name":"Microsoft.EntityFrameworkCore.Database.Command.CommandError"},"SourceContext":"Microsoft.EntityFrameworkCore.Database.Command","MachineName":"LAPTOP-TSMJLLUF","ProcessId":31520,"ThreadId":19,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:39:44.1881379+03:30","Level":"Error","MessageTemplate":"An exception occurred in the database while saving changes for context type '{contextType}'.{newline}{error}","Exception":"Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot insert the value NULL into column 'TenantId', table 'PhoenixDb.dbo.ApplicationUsers'; column does not allow nulls. INSERT fails.\r\nThe statement has been terminated.\r\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__188_0(Task`1 result)\r\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\r\n   at System.Threading.Tasks.Task.<>c.<.cctor>b__272_0(Object obj)\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location ---\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\nClientConnectionId:401fa0f2-f8c7-4807-96c2-240096ebabe9\r\nError Number:515,State:2,Class:16\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)","Properties":{"contextType":"Phoenix.DataSources.Infrastructures.DBContexts.EFDataContext","newline":"\r\n","error":"Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot insert the value NULL into column 'TenantId', table 'PhoenixDb.dbo.ApplicationUsers'; column does not allow nulls. INSERT fails.\r\nThe statement has been terminated.\r\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__188_0(Task`1 result)\r\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\r\n   at System.Threading.Tasks.Task.<>c.<.cctor>b__272_0(Object obj)\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location ---\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\nClientConnectionId:401fa0f2-f8c7-4807-96c2-240096ebabe9\r\nError Number:515,State:2,Class:16\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)","EventId":{"Id":10000,"Name":"Microsoft.EntityFrameworkCore.Update.SaveChangesFailed"},"SourceContext":"Microsoft.EntityFrameworkCore.Update","MachineName":"LAPTOP-TSMJLLUF","ProcessId":31520,"ThreadId":19,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:39:44.4828147+03:30","Level":"Fatal","MessageTemplate":"Unhandled exception","Exception":"System.AggregateException: One or more errors occurred. (An error occurred while saving the entity changes. See the inner exception for details.)\r\n ---> Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot insert the value NULL into column 'TenantId', table 'PhoenixDb.dbo.ApplicationUsers'; column does not allow nulls. INSERT fails.\r\nThe statement has been terminated.\r\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__188_0(Task`1 result)\r\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\r\n   at System.Threading.Tasks.Task.<>c.<.cctor>b__272_0(Object obj)\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location ---\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\nClientConnectionId:401fa0f2-f8c7-4807-96c2-240096ebabe9\r\nError Number:515,State:2,Class:16\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.AspNetCore.Identity.EntityFrameworkCore.UserStore`9.CreateAsync(TUser user, CancellationToken cancellationToken)\r\n   at Microsoft.AspNetCore.Identity.UserManager`1.CreateAsync(TUser user)\r\n   at Microsoft.AspNetCore.Identity.UserManager`1.CreateAsync(TUser user, String password)\r\n   at Phoenix.Application.Services.ApplicationUsers.ApplicationUserAppService.AddSuperAdminUser(ApplicationUser adminUser) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Services\\Services\\ApplicationUsers\\ApplicationUserAppService.cs:line 52\r\n   at Phoenix.Infrastructure.SeedDatas.AdminSeedDataConfiguration.UseSeedData(IApplicationBuilder applicationBuilder, IConfiguration configuration) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\SeedDatas\\AdminSeedDataConfiguration.cs:line 43\r\n   --- End of inner exception stack trace ---\r\n   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)\r\n   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)\r\n   at System.Threading.Tasks.Task.Wait()\r\n   at Phoenix.Infrastructure.Startup.UseInfrastructure(IApplicationBuilder builder, IConfiguration config) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\Startup.cs:line 109\r\n   at Program.<Main>$(String[] args) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.RestApi\\Program.cs:line 38","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":31520,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:39:44.6852164+03:30","Level":"Information","MessageTemplate":"Server Shutting down...","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":31520,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:48:28.6943651+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":13484,"ThreadId":6,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:48:32.7718000+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":13484,"ThreadId":6,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:48:51.4842383+03:30","Level":"Error","MessageTemplate":"Failed executing DbCommand ({elapsed}ms) [Parameters=[{parameters}], CommandType='{commandType}', CommandTimeout='{commandTimeout}']{newLine}{commandText}","Properties":{"elapsed":"15","parameters":"@p0='?' (Size = 450), @p1='?' (DbType = Int32), @p2='?' (Size = 4000), @p3='?' (DbType = DateTime2), @p4='?' (Size = 100), @p5='?' (DbType = Boolean), @p6='?' (Size = 100), @p7='?' (DbType = Boolean), @p8='?' (DbType = DateTimeOffset), @p9='?' (Size = 10), @p10='?' (Size = 50), @p11='?' (Size = 4000), @p12='?' (Size = 4000), @p13='?' (Size = 450), @p14='?' (DbType = Boolean), @p15='?' (Size = 50), @p16='?' (Size = 5), @p17='?' (Size = 11)","commandType":"Text","commandTimeout":30,"newLine":"\r\n","commandText":"SET NOCOUNT ON;\r\nINSERT INTO [ApplicationUsers] ([Id], [AccessFailedCount], [ConcurrencyStamp], [CreationDate], [FirstName], [IsActive], [LastName], [LockoutEnabled], [LockoutEnd], [NationalCode], [NormalizedUserName], [PasswordHash], [SecurityStamp], [TenantId], [TwoFactorEnabled], [UserName], [Mobile_CountryCallingCode], [Mobile_Number])\r\nVALUES (@p0, @p1, @p2, @p3, @p4, @p5, @p6, @p7, @p8, @p9, @p10, @p11, @p12, @p13, @p14, @p15, @p16, @p17);","EventId":{"Id":20102,"Name":"Microsoft.EntityFrameworkCore.Database.Command.CommandError"},"SourceContext":"Microsoft.EntityFrameworkCore.Database.Command","MachineName":"LAPTOP-TSMJLLUF","ProcessId":13484,"ThreadId":20,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:48:51.5155490+03:30","Level":"Error","MessageTemplate":"An exception occurred in the database while saving changes for context type '{contextType}'.{newline}{error}","Exception":"Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot insert the value NULL into column 'TenantId', table 'PhoenixDb.dbo.ApplicationUsers'; column does not allow nulls. INSERT fails.\r\nThe statement has been terminated.\r\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__188_0(Task`1 result)\r\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\r\n   at System.Threading.Tasks.Task.<>c.<.cctor>b__272_0(Object obj)\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location ---\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\nClientConnectionId:6a13243b-c24c-4777-9c94-32587609275a\r\nError Number:515,State:2,Class:16\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)","Properties":{"contextType":"Phoenix.DataSources.Infrastructures.DBContexts.EFDataContext","newline":"\r\n","error":"Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot insert the value NULL into column 'TenantId', table 'PhoenixDb.dbo.ApplicationUsers'; column does not allow nulls. INSERT fails.\r\nThe statement has been terminated.\r\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__188_0(Task`1 result)\r\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\r\n   at System.Threading.Tasks.Task.<>c.<.cctor>b__272_0(Object obj)\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location ---\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\nClientConnectionId:6a13243b-c24c-4777-9c94-32587609275a\r\nError Number:515,State:2,Class:16\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)","EventId":{"Id":10000,"Name":"Microsoft.EntityFrameworkCore.Update.SaveChangesFailed"},"SourceContext":"Microsoft.EntityFrameworkCore.Update","MachineName":"LAPTOP-TSMJLLUF","ProcessId":13484,"ThreadId":20,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:48:51.8961723+03:30","Level":"Fatal","MessageTemplate":"Unhandled exception","Exception":"System.AggregateException: One or more errors occurred. (An error occurred while saving the entity changes. See the inner exception for details.)\r\n ---> Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot insert the value NULL into column 'TenantId', table 'PhoenixDb.dbo.ApplicationUsers'; column does not allow nulls. INSERT fails.\r\nThe statement has been terminated.\r\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__188_0(Task`1 result)\r\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\r\n   at System.Threading.Tasks.Task.<>c.<.cctor>b__272_0(Object obj)\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location ---\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\nClientConnectionId:6a13243b-c24c-4777-9c94-32587609275a\r\nError Number:515,State:2,Class:16\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.AspNetCore.Identity.EntityFrameworkCore.UserStore`9.CreateAsync(TUser user, CancellationToken cancellationToken)\r\n   at Microsoft.AspNetCore.Identity.UserManager`1.CreateAsync(TUser user)\r\n   at Microsoft.AspNetCore.Identity.UserManager`1.CreateAsync(TUser user, String password)\r\n   at Phoenix.Application.Services.ApplicationUsers.ApplicationUserAppService.AddSuperAdminUser(ApplicationUser adminUser) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Services\\Services\\ApplicationUsers\\ApplicationUserAppService.cs:line 52\r\n   at Phoenix.Infrastructure.SeedDatas.AdminSeedDataConfiguration.UseSeedData(IApplicationBuilder applicationBuilder, IConfiguration configuration) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\SeedDatas\\AdminSeedDataConfiguration.cs:line 63\r\n   --- End of inner exception stack trace ---\r\n   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)\r\n   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)\r\n   at System.Threading.Tasks.Task.Wait()\r\n   at Phoenix.Infrastructure.Startup.UseInfrastructure(IApplicationBuilder builder, IConfiguration config) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\Startup.cs:line 109\r\n   at Program.<Main>$(String[] args) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.RestApi\\Program.cs:line 38","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":13484,"ThreadId":6,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:48:52.1292397+03:30","Level":"Information","MessageTemplate":"Server Shutting down...","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":13484,"ThreadId":6,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:51:27.9491032+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":14552,"ThreadId":6,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:51:31.4543507+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":14552,"ThreadId":6,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:51:39.2872070+03:30","Level":"Fatal","MessageTemplate":"Unhandled exception","Exception":"System.AggregateException: One or more errors occurred. (Exception of type 'Phoenix.Application.Services.ApplicationUsers.Exceptions.CreateUserFailedException' was thrown.)\r\n ---> Phoenix.Application.Services.ApplicationUsers.Exceptions.CreateUserFailedException: Exception of type 'Phoenix.Application.Services.ApplicationUsers.Exceptions.CreateUserFailedException' was thrown.\r\n   at Phoenix.Application.Services.ApplicationUsers.ApplicationUserAppService.GuardAgainstFailedRegisteration(IdentityResult processResult) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Services\\Services\\ApplicationUsers\\ApplicationUserAppService.cs:line 216\r\n   at Phoenix.Application.Services.ApplicationUsers.ApplicationUserAppService.AddSuperAdminUser(ApplicationUser adminUser) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Services\\Services\\ApplicationUsers\\ApplicationUserAppService.cs:line 44\r\n   at Phoenix.Infrastructure.SeedDatas.AdminSeedDataConfiguration.UseSeedData(IApplicationBuilder applicationBuilder, IConfiguration configuration) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\SeedDatas\\AdminSeedDataConfiguration.cs:line 63\r\n   --- End of inner exception stack trace ---\r\n   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)\r\n   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)\r\n   at System.Threading.Tasks.Task.Wait()\r\n   at Phoenix.Infrastructure.Startup.UseInfrastructure(IApplicationBuilder builder, IConfiguration config) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\Startup.cs:line 109\r\n   at Program.<Main>$(String[] args) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.RestApi\\Program.cs:line 38","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":14552,"ThreadId":6,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:51:39.3778203+03:30","Level":"Information","MessageTemplate":"Server Shutting down...","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":14552,"ThreadId":6,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:52:11.7452159+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":25500,"ThreadId":10,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:52:15.0408139+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":25500,"ThreadId":10,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:52:20.1736342+03:30","Level":"Fatal","MessageTemplate":"Unhandled exception","Exception":"System.AggregateException: One or more errors occurred. (Exception of type 'Phoenix.Application.Services.ApplicationUsers.Exceptions.CreateUserFailedException' was thrown.)\r\n ---> Phoenix.Application.Services.ApplicationUsers.Exceptions.CreateUserFailedException: Exception of type 'Phoenix.Application.Services.ApplicationUsers.Exceptions.CreateUserFailedException' was thrown.\r\n   at Phoenix.Application.Services.ApplicationUsers.ApplicationUserAppService.GuardAgainstFailedRegisteration(IdentityResult processResult) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Services\\Services\\ApplicationUsers\\ApplicationUserAppService.cs:line 216\r\n   at Phoenix.Application.Services.ApplicationUsers.ApplicationUserAppService.AddSuperAdminUser(ApplicationUser adminUser) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Services\\Services\\ApplicationUsers\\ApplicationUserAppService.cs:line 44\r\n   at Phoenix.Infrastructure.SeedDatas.AdminSeedDataConfiguration.UseSeedData(IApplicationBuilder applicationBuilder, IConfiguration configuration) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\SeedDatas\\AdminSeedDataConfiguration.cs:line 63\r\n   --- End of inner exception stack trace ---\r\n   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)\r\n   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)\r\n   at System.Threading.Tasks.Task.Wait()\r\n   at Phoenix.Infrastructure.Startup.UseInfrastructure(IApplicationBuilder builder, IConfiguration config) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\Startup.cs:line 109\r\n   at Program.<Main>$(String[] args) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.RestApi\\Program.cs:line 38","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":25500,"ThreadId":10,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:52:20.1941362+03:30","Level":"Information","MessageTemplate":"Server Shutting down...","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":25500,"ThreadId":10,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:52:38.9627814+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":27852,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:52:42.5450453+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":27852,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:54:36.5958311+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":29728,"ThreadId":14,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:54:40.2353975+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":29728,"ThreadId":14,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:56:10.4628959+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":19100,"ThreadId":14,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:56:13.8217104+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":19100,"ThreadId":14,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:56:16.2619926+03:30","Level":"Fatal","MessageTemplate":"Unhandled exception","Exception":"System.AggregateException: One or more errors occurred. (Exception of type 'Phoenix.Application.Services.ApplicationUsers.Exceptions.CreateUserFailedException' was thrown.)\r\n ---> Phoenix.Application.Services.ApplicationUsers.Exceptions.CreateUserFailedException: Exception of type 'Phoenix.Application.Services.ApplicationUsers.Exceptions.CreateUserFailedException' was thrown.\r\n   at Phoenix.Application.Services.ApplicationUsers.ApplicationUserAppService.GuardAgainstFailedRegisteration(IdentityResult processResult) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Services\\Services\\ApplicationUsers\\ApplicationUserAppService.cs:line 216\r\n   at Phoenix.Application.Services.ApplicationUsers.ApplicationUserAppService.AddSuperAdminUser(ApplicationUser adminUser) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Services\\Services\\ApplicationUsers\\ApplicationUserAppService.cs:line 44\r\n   at Phoenix.Infrastructure.SeedDatas.AdminSeedDataConfiguration.UseSeedData(IApplicationBuilder applicationBuilder, IConfiguration configuration) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\SeedDatas\\AdminSeedDataConfiguration.cs:line 63\r\n   --- End of inner exception stack trace ---\r\n   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)\r\n   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)\r\n   at System.Threading.Tasks.Task.Wait()\r\n   at Phoenix.Infrastructure.Startup.UseInfrastructure(IApplicationBuilder builder, IConfiguration config) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\Startup.cs:line 109\r\n   at Program.<Main>$(String[] args) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.RestApi\\Program.cs:line 38","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":19100,"ThreadId":14,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:56:16.3937958+03:30","Level":"Information","MessageTemplate":"Server Shutting down...","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":19100,"ThreadId":14,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:56:33.7844823+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":26164,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:56:37.3065952+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":26164,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:56:41.5707786+03:30","Level":"Fatal","MessageTemplate":"Unhandled exception","Exception":"System.AggregateException: One or more errors occurred. (Exception of type 'Phoenix.Application.Services.ApplicationUsers.Exceptions.CreateUserFailedException' was thrown.)\r\n ---> Phoenix.Application.Services.ApplicationUsers.Exceptions.CreateUserFailedException: Exception of type 'Phoenix.Application.Services.ApplicationUsers.Exceptions.CreateUserFailedException' was thrown.\r\n   at Phoenix.Application.Services.ApplicationUsers.ApplicationUserAppService.GuardAgainstFailedRegisteration(IdentityResult processResult) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Services\\Services\\ApplicationUsers\\ApplicationUserAppService.cs:line 216\r\n   at Phoenix.Application.Services.ApplicationUsers.ApplicationUserAppService.AddSuperAdminUser(ApplicationUser adminUser) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Services\\Services\\ApplicationUsers\\ApplicationUserAppService.cs:line 44\r\n   at Phoenix.Infrastructure.SeedDatas.AdminSeedDataConfiguration.UseSeedData(IApplicationBuilder applicationBuilder, IConfiguration configuration) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\SeedDatas\\AdminSeedDataConfiguration.cs:line 63\r\n   --- End of inner exception stack trace ---\r\n   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)\r\n   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)\r\n   at System.Threading.Tasks.Task.Wait()\r\n   at Phoenix.Infrastructure.Startup.UseInfrastructure(IApplicationBuilder builder, IConfiguration config) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\Startup.cs:line 109\r\n   at Program.<Main>$(String[] args) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.RestApi\\Program.cs:line 38","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":26164,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:56:41.5985723+03:30","Level":"Information","MessageTemplate":"Server Shutting down...","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":26164,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:57:09.2269190+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":22732,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:57:12.6671653+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":22732,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:58:18.3351867+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":15040,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:58:21.8182918+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":15040,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:58:53.4378992+03:30","Level":"Fatal","MessageTemplate":"Unhandled exception","Exception":"System.AggregateException: One or more errors occurred. (Exception of type 'Phoenix.Application.Services.ApplicationUsers.Exceptions.CreateUserFailedException' was thrown.)\r\n ---> Phoenix.Application.Services.ApplicationUsers.Exceptions.CreateUserFailedException: Exception of type 'Phoenix.Application.Services.ApplicationUsers.Exceptions.CreateUserFailedException' was thrown.\r\n   at Phoenix.Application.Services.ApplicationUsers.ApplicationUserAppService.GuardAgainstFailedRegisteration(IdentityResult processResult) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Services\\Services\\ApplicationUsers\\ApplicationUserAppService.cs:line 216\r\n   at Phoenix.Application.Services.ApplicationUsers.ApplicationUserAppService.AddSuperAdminUser(ApplicationUser adminUser) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Services\\Services\\ApplicationUsers\\ApplicationUserAppService.cs:line 44\r\n   at Phoenix.Infrastructure.SeedDatas.AdminSeedDataConfiguration.UseSeedData(IApplicationBuilder applicationBuilder, IConfiguration configuration) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\SeedDatas\\AdminSeedDataConfiguration.cs:line 61\r\n   --- End of inner exception stack trace ---\r\n   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)\r\n   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)\r\n   at System.Threading.Tasks.Task.Wait()\r\n   at Phoenix.Infrastructure.Startup.UseInfrastructure(IApplicationBuilder builder, IConfiguration config) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.Infrastructure\\Startup.cs:line 109\r\n   at Program.<Main>$(String[] args) in D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.RestApi\\Program.cs:line 38","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":15040,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:58:53.4649070+03:30","Level":"Information","MessageTemplate":"Server Shutting down...","Properties":{"MachineName":"LAPTOP-TSMJLLUF","ProcessId":15040,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:59:10.7480097+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":20780,"ThreadId":12,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T00:59:14.4685201+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":20780,"ThreadId":12,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:01:12.5608681+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":22452,"ThreadId":6,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:01:15.9692768+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":22452,"ThreadId":6,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:03:33.2691408+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":16064,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:03:36.6559770+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":16064,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:03:59.7328832+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":21708,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:04:03.2280993+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":21708,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:04:07.7481437+03:30","Level":"Information","MessageTemplate":"Now listening on: {address}","Properties":{"address":"http://localhost:5000","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime","MachineName":"LAPTOP-TSMJLLUF","ProcessId":21708,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:04:07.7513003+03:30","Level":"Information","MessageTemplate":"Application started. Press Ctrl+C to shut down.","Properties":{"SourceContext":"Microsoft.Hosting.Lifetime","MachineName":"LAPTOP-TSMJLLUF","ProcessId":21708,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:04:07.7515068+03:30","Level":"Information","MessageTemplate":"Hosting environment: {envName}","Properties":{"envName":"Development","SourceContext":"Microsoft.Hosting.Lifetime","MachineName":"LAPTOP-TSMJLLUF","ProcessId":21708,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:04:07.7516406+03:30","Level":"Information","MessageTemplate":"Content root path: {contentRoot}","Properties":{"contentRoot":"D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.RestApi\\","SourceContext":"Microsoft.Hosting.Lifetime","MachineName":"LAPTOP-TSMJLLUF","ProcessId":21708,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:05:42.9498563+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":4572,"ThreadId":9,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:06:22.8563945+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":24584,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:06:29.3073495+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":24584,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:06:33.1872619+03:30","Level":"Information","MessageTemplate":"Now listening on: {address}","Properties":{"address":"http://localhost:5000","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime","MachineName":"LAPTOP-TSMJLLUF","ProcessId":24584,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:06:33.1900062+03:30","Level":"Information","MessageTemplate":"Application started. Press Ctrl+C to shut down.","Properties":{"SourceContext":"Microsoft.Hosting.Lifetime","MachineName":"LAPTOP-TSMJLLUF","ProcessId":24584,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:06:33.1906341+03:30","Level":"Information","MessageTemplate":"Hosting environment: {envName}","Properties":{"envName":"Development","SourceContext":"Microsoft.Hosting.Lifetime","MachineName":"LAPTOP-TSMJLLUF","ProcessId":24584,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:06:33.2032942+03:30","Level":"Information","MessageTemplate":"Content root path: {contentRoot}","Properties":{"contentRoot":"D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.RestApi\\","SourceContext":"Microsoft.Hosting.Lifetime","MachineName":"LAPTOP-TSMJLLUF","ProcessId":24584,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:12:06.4916594+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":5884,"ThreadId":6,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:12:11.3808907+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":5884,"ThreadId":6,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:12:13.8686410+03:30","Level":"Information","MessageTemplate":"Now listening on: {address}","Properties":{"address":"http://localhost:5000","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime","MachineName":"LAPTOP-TSMJLLUF","ProcessId":5884,"ThreadId":6,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:12:13.8699505+03:30","Level":"Information","MessageTemplate":"Application started. Press Ctrl+C to shut down.","Properties":{"SourceContext":"Microsoft.Hosting.Lifetime","MachineName":"LAPTOP-TSMJLLUF","ProcessId":5884,"ThreadId":6,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:12:13.8701498+03:30","Level":"Information","MessageTemplate":"Hosting environment: {envName}","Properties":{"envName":"Development","SourceContext":"Microsoft.Hosting.Lifetime","MachineName":"LAPTOP-TSMJLLUF","ProcessId":5884,"ThreadId":6,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:12:13.8702619+03:30","Level":"Information","MessageTemplate":"Content root path: {contentRoot}","Properties":{"contentRoot":"D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.RestApi\\","SourceContext":"Microsoft.Hosting.Lifetime","MachineName":"LAPTOP-TSMJLLUF","ProcessId":5884,"ThreadId":6,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:15:54.8586411+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":32360,"ThreadId":10,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:16:17.3525451+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":31816,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:16:21.1984516+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":31816,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:16:24.0449731+03:30","Level":"Information","MessageTemplate":"Now listening on: {address}","Properties":{"address":"http://localhost:5000","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime","MachineName":"LAPTOP-TSMJLLUF","ProcessId":31816,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:16:24.0463706+03:30","Level":"Information","MessageTemplate":"Application started. Press Ctrl+C to shut down.","Properties":{"SourceContext":"Microsoft.Hosting.Lifetime","MachineName":"LAPTOP-TSMJLLUF","ProcessId":31816,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:16:24.0467091+03:30","Level":"Information","MessageTemplate":"Hosting environment: {envName}","Properties":{"envName":"Development","SourceContext":"Microsoft.Hosting.Lifetime","MachineName":"LAPTOP-TSMJLLUF","ProcessId":31816,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:16:24.0588025+03:30","Level":"Information","MessageTemplate":"Content root path: {contentRoot}","Properties":{"contentRoot":"D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.RestApi\\","SourceContext":"Microsoft.Hosting.Lifetime","MachineName":"LAPTOP-TSMJLLUF","ProcessId":31816,"ThreadId":13,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:18:04.0520645+03:30","Level":"Warning","MessageTemplate":"{State:l}","Properties":{"State":"Stopping the server due to DomainUnload or ProcessExit event...","SourceContext":"Hangfire.Server.BackgroundProcessingServer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":31816,"ThreadId":2,"Application":"Phoenix.Web"},"Renderings":{"State":[{"Format":"l","Rendering":"Stopping the server due to DomainUnload or ProcessExit event..."}]}}
{"Timestamp":"2023-02-04T01:18:04.0558354+03:30","Level":"Warning","MessageTemplate":"{State:l}","Properties":{"State":"Server a3e31708-20fb-402b-971b-f5c0d84d4106 caught shutdown signal...","SourceContext":"Hangfire.Server.BackgroundServerProcess","MachineName":"LAPTOP-TSMJLLUF","ProcessId":31816,"ThreadId":2,"Application":"Phoenix.Web"},"Renderings":{"State":[{"Format":"l","Rendering":"Server a3e31708-20fb-402b-971b-f5c0d84d4106 caught shutdown signal..."}]}}
{"Timestamp":"2023-02-04T01:18:04.0910099+03:30","Level":"Warning","MessageTemplate":"{State:l}","Properties":{"State":"Server a3e31708-20fb-402b-971b-f5c0d84d4106 stopped non-gracefully due to ServerWatchdog, ServerJobCancellationWatcher, ExpirationManager, CountersAggregator, SqlServerHeartbeatProcess, Worker, DelayedJobScheduler, RecurringJobScheduler. Outstanding work on those dispatchers could be aborted, and there can be delays in background processing. This server instance will be incorrectly shown as active for a while. To avoid non-graceful shutdowns, investigate what prevents from stopping gracefully and add CancellationToken support for those methods.","SourceContext":"Hangfire.Server.BackgroundServerProcess","MachineName":"LAPTOP-TSMJLLUF","ProcessId":31816,"ThreadId":24,"Application":"Phoenix.Web"},"Renderings":{"State":[{"Format":"l","Rendering":"Server a3e31708-20fb-402b-971b-f5c0d84d4106 stopped non-gracefully due to ServerWatchdog, ServerJobCancellationWatcher, ExpirationManager, CountersAggregator, SqlServerHeartbeatProcess, Worker, DelayedJobScheduler, RecurringJobScheduler. Outstanding work on those dispatchers could be aborted, and there can be delays in background processing. This server instance will be incorrectly shown as active for a while. To avoid non-graceful shutdowns, investigate what prevents from stopping gracefully and add CancellationToken support for those methods."}]}}
{"Timestamp":"2023-02-04T01:36:10.6339513+03:30","Level":"Information","MessageTemplate":"Applying Root Migrations ...","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":18896,"ThreadId":9,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:36:14.5531962+03:30","Level":"Information","MessageTemplate":"\r\n*******  | DataBase Migration Updated Successfully | *******\r\n","Properties":{"SourceContext":"Phoenix.Infrastructure.Persistence.Initialization.DatabaseInitializer","MachineName":"LAPTOP-TSMJLLUF","ProcessId":18896,"ThreadId":9,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:36:17.9598308+03:30","Level":"Information","MessageTemplate":"Now listening on: {address}","Properties":{"address":"http://localhost:5000","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime","MachineName":"LAPTOP-TSMJLLUF","ProcessId":18896,"ThreadId":9,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:36:17.9615195+03:30","Level":"Information","MessageTemplate":"Application started. Press Ctrl+C to shut down.","Properties":{"SourceContext":"Microsoft.Hosting.Lifetime","MachineName":"LAPTOP-TSMJLLUF","ProcessId":18896,"ThreadId":9,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:36:17.9617112+03:30","Level":"Information","MessageTemplate":"Hosting environment: {envName}","Properties":{"envName":"Development","SourceContext":"Microsoft.Hosting.Lifetime","MachineName":"LAPTOP-TSMJLLUF","ProcessId":18896,"ThreadId":9,"Application":"Phoenix.Web"}}
{"Timestamp":"2023-02-04T01:36:17.9618088+03:30","Level":"Information","MessageTemplate":"Content root path: {contentRoot}","Properties":{"contentRoot":"D:\\MyArchitecture\\Main\\SolvedProblems\\dotnet-phoenix-clean-architecture\\src\\Phoenix.RestApi\\","SourceContext":"Microsoft.Hosting.Lifetime","MachineName":"LAPTOP-TSMJLLUF","ProcessId":18896,"ThreadId":9,"Application":"Phoenix.Web"}}
